packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -137,9 +135,9 @@ export function computePerformanceResourceDetails(
   return details
 }
 
-export function toValidEntry(entry: RumPerformanceResourceTiming) {
+export function isValidEntry(entry: RumPerformanceResourceTiming) {
```

Comment:
I renamed this as I believe it's more correct because 
- it's not transforming the entry. 
- it is only used in a `if` condition and in a `Array.filter()` method